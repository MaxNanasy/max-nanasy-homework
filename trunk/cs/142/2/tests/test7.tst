var a, b, c : array 10 of short;
main()
{
a[1] = b[2] + c[3] - d[4] * 5 / 7;
print(a);
}