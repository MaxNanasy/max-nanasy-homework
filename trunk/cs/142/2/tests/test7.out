program
   declarations
      newType
         type
   statementSequence
      statement
         assignment
            designator
               expression
                  term
                     factor
            expression
               term
                  factor
                     designator
                        expression
                           term
                              factor
               term
                  factor
                     designator
                        expression
                           term
                              factor
               term
                  factor
                     designator
                        expression
                           term
                              factor
                  factor
                  factor
      statement
         output
            parameters
               designator
